tasties_app/models.py

```diff
@@ -26,6 +38,46 @@ class Recipe(models.Model):
     minutes_to_make = models.IntegerField(validators=[MinValueValidator(1)])
     recipe_picture = models.ImageField(upload_to="recipe_pictures")
 
+    def edit_recipe(self, new_title, new_description,
```

Comment:
We talked about the CRUD functions that django takes care of, so maybe this method is unnecessary anymore.